src/data/index.ts

```diff
@@ -26,7 +26,11 @@ export type {
   RecordMetadataValue,
 } from './types';
 export { PineconeConfigurationSchema } from './types';
-export type { DeleteManyOptions } from './deleteMany';
+export type {
+  DeleteManyOptions,
+  DeleteManyByFilterOptions,
+  DeleteManyByVectorIdOptions,
```

Comment:
Can we rename `DeleteManyByVectorIdOptions` to `DeleteManyByRecordIdOptions` real quick?

Reply:
Yup! Lemme update.